Form-Controls/index.html

```diff
@@ -14,15 +14,80 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
+    <div>
+      <label>Name</label>
+      <input name="Name" id="name" required>
+    </div>
+    <div>
+      <label>Email</label>
+      <input type="email" name="Email" id="Email" required>
+    </div>
+    
+    <div>
+      <label>T-Shirt Colour</label>
+      <select>
+        <option>
+          Blue
+      
+        </option>
+        <option>
+          Red
+        </option>
+        <option>
+          Yellow
+        </option>
+      </select>
+    </div>
```

Comment:
The `for` attribute is the same here, but you need to give the `select` element an id first.
`<label for="tshirt-colour">T-Shirt Colour</label>`
      `<select id="tshirt-colour">`